processing/src/main/java/io/druid/query/groupby/GroupByQueryEngine.java

```diff
@@ -71,6 +79,62 @@
   private final Supplier<GroupByQueryConfig> config;
   private final StupidPool<ByteBuffer> intermediateResultsBufferPool;
 
+  private static void getBytesFromBuffer(ByteBuffer src, byte[] dst, int srcOffset, int readLen)
```

Comment:
I had structured this to do absolute reads instead of relative, and I didn't see an absolute bulk get method for ByteBuffer. Looking at this again, I don't see a reason not to do relative, so I'll get rid of this